runtime/runc_jailer.go

```diff
@@ -41,8 +41,19 @@ import (
 
 const (
 	networkNamespaceRuncName = "network"
+	cacheTopologyPath        = "/sys/devices/system/cpu/cpu0/cache"
+	cacheFolderPrefix        = "index"
 )
 
+var cacheTopologyPaths = []string{
+	"sys",
+	"devices",
+	"system",
+	"cpu",
+	"cpu0",
+	"cache",
+}
```

Comment:
- Does the directory has `cpu0` under `cpu0/cache`?
- What we copy is different from what Firecracker's jailer copies in https://github.com/firecracker-microvm/firecracker/pull/2334. Do you know the reason? (cc @dianpopa)

Reply:
Yea, I asked Diana and she told me today what files to copy. And yes, all these directories will exist.